spec/noughts_and_crosses_spec.rb

```diff
@@ -65,6 +69,20 @@
 
         expect(game.winner).to eq nil
       end
+
+      it "is full when all spots have been taken" do
+        game.take_turn(4)
+        game.take_turn(1)
+        game.take_turn(5)
+        game.take_turn(3)
+        game.take_turn(2)
+        game.take_turn(8)
+        game.take_turn(0)
+        game.take_turn(7)
+        game.take_turn(6)
+
+        expect(game.board_full?).to eq true
```

Comment:
How do you know this won't return true one move before the end or when the board is empty?